packages/@tinacms/api-git/src/router.ts

```diff
@@ -84,9 +84,21 @@ async function createSSHKey(pathRoot: string) {
       encoding: 'utf8',
       mode: 0o600,
     })
+  }
+}
 
-    updateRemoteToSSH(pathRoot)
+async function configureGitRemote(pathRoot: string) {
+  await createSSHKey(pathRoot)
+  if (process.env.GIT_REMOTE) {
+    await updateOrigin(pathRoot, process.env.GIT_REMOTE)
```

Comment:
We should log a warning message if there's already an existing `origin` that we're overwriting